spec/type.dd

```diff
@@ -119,8 +119,8 @@ $(H2 $(LEGACY_LNAME2 Basic Data Types, basic-data-types, Basic Data Types))
     $(TROW $(D cdouble), $(D double.nan+double.nan*1.0i), complex double)
     $(TROW $(D creal), $(D real.nan+real.nan*1.0i), complex real)
     $(TROW $(D char), $(D '\xFF'), unsigned 8 bit (UTF-8 code unit))
-    $(TROW $(D wchar), $(D '\uFFFF'), unsigned 16 bit (UTF-16 code unit))
-    $(TROW $(D dchar), $(D '\U0000FFFF'), unsigned 32 bit (UTF-32 code unit))
+    $(TROW $(D wchar), $(D '\uFFFF'), unsigned 16 bit (UTF-16 code unit) matching the target endianness)
```

Comment:
> target 
Does that cover CTFE?

Reply:
CTFE does not expose endianness.

Reply:
Why not say it for all?  Do `ushort`'s match the target endianness?  What about `double`'s?

I'm being pedantic, but I think leaving this detail for abi.dd is enough.